modules/gitops-configure-rollout-route-traffic-using-openshift-routes.adoc

```diff
@@ -0,0 +1,282 @@
+<2> Ensure that the contents of the `selector` field are the same as in stable service and `Rollout` CR 
++
+[IMPORTANT]
+====
+Ensure that the name of the canary service specified in the `Route` object matches with the name of the canary service specified in the `Service` object.
+====
+.. Click *Create* to create the canary service.
++
+Rollouts automatically update the created service with pod template hash of the canary `ReplicaSet`. For example, `rollouts-pod-template-hash: 7bf84f9696`.
+.. Similarly, repeat these steps to create the stable service:
++
+The following example creates a stable service called `argo-rollouts-stable-service`. Stable traffic is directed to this service.
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Service
+metadata:
+  name: argo-rollouts-stable-service
+spec:
+  ports: #<1>
+  - port: 80
+    targetPort: http
+    protocol: TCP
+    name: http
+
+  selector: #<2> 
+    app: rollouts-demo
+----
+<1> Specifies the name of the port used by the application for running inside the container.
+<2> Ensure that the contents of the `selector` field are the same as in canary service and `Rollout` CR .
++
+[IMPORTANT]
+====
+Ensure that the name of the stable service specified in the `Route` object matches with the name of the stable service specified in the `Service` object.
+====
+.. Click *Create* to create the stable service.
++
+Rollouts automatically update the created service with pod template hash of the stable `ReplicaSet`. For example, `rollouts-pod-template-hash: 1b6a7733`.
+. Create the `Rollout` CR to reference the `Route` and `Service` objects.
+.. In the *Administrator* perspective of the web console, go to *Operators* -> *Installed Operators* -> *Red Hat OpenShift GitOps* -> *Rollout*.
+.. On the *Create Rollout* page, click *YAML view* and add the following snippet:
++
+The following example creates a `Rollout` CR called `rollouts-demo`:
++
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: Rollout
+metadata:
+  name: rollouts-demo
+spec:
+  template:
+    metadata:
+      labels:
+        app: rollouts-demo
+    spec:
+      containers: #<1>
+      - name: rollouts-demo
+        image: argoproj/rollouts-demo:blue
+        ports:
+        - name: http
+          containerPort: 8080
+          protocol: TCP
+        resources:
+          requests:
+            memory: 32Mi
+            cpu: 5m
+                
+  revisionHistoryLimit: 2
+  replicas: 5
+  strategy:
+    canary:
+      canaryService: argo-rollouts-canary-service #<2>
+      stableService: argo-rollouts-stable-service #<3>
+      trafficRouting:
+        plugins:
+          argoproj-labs/openshift:          
+            routes: #<3>
+              - rollouts-demo-route  #<4>
+      steps: #<5>
+      - setWeight: 30
+      - pause: {}
+      - setWeight: 60
+      - pause: {}
+  selector: #<6>
+    matchLabels:
+      app: rollouts-demo
+----
+<1> Specifies the name of the container used in the rollout.
```

Comment:
I found this handy example YAML that adds more information: https://argo-rollouts.readthedocs.io/en/stable/features/specification/
So I would sugest moving callout <1> to the template:
```suggestion
<1> Specifies the pods that are to be created. 
```